### ISSUE TITLE
When widget is created it fires change output event. 

### ISSUE THREAD

**Issue by speculees** [swiety85/angular2gridster#186]
Hello,
Here is my scenario. I receive an array of widgets from some endpoint. When user changes the widget position or size - the update endpoint is called, and the "change" output emit is perfect for that. However, as we initially loop trough the received widgets list, each widget emits on "change" output, creates an update on the BE.

`<gridster-item *ngFor="let widget of dashboard.widgets; let indx = index" [options]="gridsterItemOptions" [(x)]="widget.x"
            [(y)]="widget.y" [xSm]="0" [xSm]="0" [(xMd)]="widget.x" [(yMd)]="widget.y" [(xLg)]="widget.x" [(yLg)]="widget.y"
            [(xXl)]="widget.x" [(yXl)]="widget.y" [(w)]="widget.w" [(h)]="widget.h" (change)="onGridsterChange($event)">`

To recap again - after every reference change in **let widget** the **onGridsterChange** event is called

**Comment by speculees** [comment]
Perhaps if "gridster" component would have an input [gridster]="dashboard", and gridster-item becomes an ng-template with let-widget, then it would be easy to distinguish between initial dashboard state and adding a new widget...

**Comment by swiety85** [comment]
Hi,
In your example code I see that you assign the same property to coords for every breakpoint. This will generate an issue like you said. Gridster calculates the items position for each breakpoint, even if it's rendered in one specific window width and if the position for specific breakpoint is invalid it overwrites it. That's why this event is called initially. 
If you don't need different items positions for different window sizes then you should set `responsiveView=false`, remove `responsiveOptions` and use only `[(x)]` and  `[(y)]` for coords binding.
If you need responsive behaviour, then your `widget` object should have dedicated coord property for each breakpoint (like it's done in [demo](https://github.com/swiety85/angular2gridster/blob/master/demo/src/app/app.component.html#L93)).
Both solution should work and not call change event initially, if nothing has changed.

Hope I help you somehow.

**Comment by speculees** [comment]
You did and so did 1.2.1 update :+1: 
Thank you!
